src/rgw/rgw_pubsub_push.cc

```diff
@@ -461,6 +736,10 @@ RGWPubSubEndpoint::Ptr RGWPubSubEndpoint::create(const std::string& endpoint,
   } else if (schema == "amqps") {
     throw configuration_error("AMQP: ssl not supported");
     return nullptr;
+#endif
+#ifdef WITH_RADOSGW_KAFKA_ENDPOINT
+  } else if (schema == KAFKA_SCHEMA) {
```

Comment:
@yuvalif I nautilus releases I can't find this line so kafka endpoint schema won't find! Am I wrong?

Reply:
lines 740 to 743 are there. 
Did you mean that the these lines are macroed out? 
This can happen if you are using ubuntu16.04 (please see release notes).
In this case `WITH_RADOSGW_KAFKA_ENDPOINT` will always be set to `OFF`

Reply:
I don't see them in nautilus 14.2.8:
https://github.com/ceph/ceph/blob/2d095e947a02261ce61424021bb43bd3022d35cb/src/rgw/rgw_pubsub_push.cc#L464

Reply:
@clwluvw was merged to the master nautilus branch, so probably going to be in 14.2.9

Reply:
Yes I think it should be in 14.2.5 and newer.
Thanks :)

Reply:
As it appears it did not make it to 14.2.9, will it be in 14.2.10? And when would that be?

Reply:
its already merged to nautilus head, so, if it did not made it to 14.2.9 it will be in 14.2.10